fixtures/showtec/explorer-250-wash-pro.json

```diff
@@ -0,0 +1,684 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Explorer 250 Wash",
+  "categories": ["Color Changer", "Moving Head"],
+  "meta": {
+    "authors": ["Luke Nelson"],
+    "createDate": "2023-04-29",
+    "lastModifyDate": "2023-04-29",
+    "importPlugin": {
+      "plugin": "qlcplus_4.12.1",
+      "date": "2023-03-25",
+      "comment": "created by Q Light Controller Plus (version 4.12.6)"
+    }
+  },
+  "links": {
+    "manual": [
+      "https://www.manualslib.com/manual/837961/Showtec-Explorer-250-Wash.html",
+      "https://www.highlite.com/media/attachments/MANUAL/40178_MANUAL_D_V1.pdf"
```

Comment:
Looking back at the manuals, the second link is not in English. Is it worth keeping both?
 "https://www.manualslib.com/manual/837961/Showtec-Explorer-250-Wash.html"

Reply:
If possible, we only link to one English (or multi-language) manual. But manuals from the official manufacturer website are better than from those manual collecting sites. Maybe we can find the English manual there? Otherwise, let's only keep the English one from manualslib.com.

Reply:
I've had another look and can't find an English / multi-language model linked on their site. I'll remove the second link.